frontend/src/lib/components/VersionChecker/VersionCheckerBanner.tsx

```diff
@@ -2,11 +2,13 @@ import { useValues } from 'kea'
 import { LemonBanner } from 'lib/lemon-ui/LemonBanner'
 import { versionCheckerLogic } from './versionCheckerLogic'
 
-export function VersionCheckerBanner(): JSX.Element {
+export function VersionCheckerBanner({ minVersionAccepted }: { minVersionAccepted?: string }): JSX.Element {
     const { versionWarning } = useValues(versionCheckerLogic)
-
     // We don't want to show a message if the diff is too small (we might be still deploying the changes out)
-    if (!versionWarning || versionWarning.diff < 5) {
+    if (
+        !versionWarning ||
+        (minVersionAccepted ? versionWarning.currentVersion >= minVersionAccepted : versionWarning.diff < 5)
```

Comment:
makes sense!

use localeCompare for an appropriate comparison: https://github.com/PostHog/posthog/blob/master/frontend/src/lib/components/VersionChecker/versionCheckerLogic.ts#L96